libsawtooth/src/client/rest.rs

```diff
@@ -37,12 +37,14 @@ pub use super::error::SawtoothClientError;
 /// communication with the REST API.
 pub struct RestApiSawtoothClient {
     url: String,
+    auth: Option<String>,
 }
 
 /// Builder for building a RestApiSawtoothClient
 #[derive(Default)]
 pub struct RestApiSawtoothClientBuilder {
     url: Option<String>,
+    auth: Option<String>,
```

Comment:
What does this string look like?

Reply:
'username:password'

Reply:
We should not require the caller to know the format requirements. This should probably be username and password in the builder. 

We probably will want to expand this into a richer enum, but we can do that when we add more auth options